src/app/places/places.component.scss

```diff
@@ -1,3 +1,42 @@
-.images {
-  width: 300px;
+.place-card {
+  max-width: 600px;
+}
+
+.footer {
+  text-align: center;
+}
+
+.places {
+  list-style-type: none;
+}
+
+li {
+  margin: 15px 25px;
+}
+
+.info {
+  font-size: smaller;
+  font-weight: bold;
+}
+
+.description {
+  font-size: small;
+  padding-bottom: 20px;
+}
+
+.tag {
+  font-size: smaller;
+  min-height: 22px !important;
```

Comment:
`!important` should be used only as last resort :) Do we have any other options?

Reply:
Removed